### PR TITLE
SOL-390: Added ability to create a dashboard in the admin panel

### PR DESCRIPTION
**Pull request by rohan-bes**
### Issue SOL-390:

Not 100% sure I've got the permissions model right here, might do a double check later

### CODE REVIEW

**Comment by edmofro** [comment]
@rohan-bes could you chuck a screenshot or two on the ticket?

**Review by EMcQ-BES — APPROVED** [review]
Been a while since I was really in the trenches with permissions but the structure looks great! One of your unit tests isn't _quite_ testing what it says it is, but otherwise this all looks good to me.

File: packages/meditrak-server/src/tests/routes/dashboards/CreateDashboards.test.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+            code: 'some_code',
+            name: 'Some Name',
+            root_entity_code: entityCode,
+          },
+        });
+
+        expect(result).to.have.keys('error');
+      });
+
+      it('Throw an exception when trying to create a dashboard to an entity when we lack BES admin access to that entity', async () => {
```
[the remark this answers is not in the document]
Comment (rohan-bes) [reply packages/meditrak-server/src/tests/routes/dashboards/CreateDashboards.test.js:70]: Ah thanks! Nice catch 👍

**Comment by rohan-bes** [comment]
Just wanna congratulate you @EMcQ-BES and @edmofro for the excellent route and permissions architecture in the meditrak-server! It was very easy to follow and the code basically wrote itself!!

**Comment by edmofro** [comment]
All @EMcQ-BES! 🎉

**Pull request merged**